apps/api/src/campaign/campaign.service.ts

```diff
@@ -463,6 +481,8 @@ export class CampaignService {
             person: { connect: { id: paymentData.personId } },
           },
         })
+
+        this.webSocketService.sendNotification('successfulDonation', donation)
```

Comment:
here the donation is returned from a create statement and it will contain all donation fields. Could you please Pick<> only the needed fields